### PR TITLE
dcos-monitoring: Update docs for 0.2.1

### PR DESCRIPTION
**Pull request by gracedo**
## Description
Documentation for dcos-monitoring package v0.2.1 release - only change in this patch release is updating the docs to have the correct set of permissions required to run the package

## Urgency
- [ ] Blocker 
- [ ] High
- [x] Medium

## Requirements
- Test all commands and procedures.
- Add [redirects](https://github.com/mesosphere/dcos-docs-site/wiki/Redirects).
- Change all affected versions (e.g. 1.7, 1.8, 1.9, 1.10, 1.11, 1.12).
- See the [contribution guidelines](https://github.com/mesosphere/dcos-docs-site/wiki/Contributing).

### CODE REVIEW

**Comment by jieyu** [comment]
@SarahStegall thanks for the editing.

@gracedo can you sync those changes back to the dcos-monitoring repo (both master and v0.2.x branch)? We don't need to cut a new release.

Moving forward, such changes should happen in dcos-monitoring repo.

File: pages/services/dcos-monitoring/0.2.1/install/index.md [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+
+## Download the package
+
+<!-- Does this require that the user have a service account? Also, please add a link to the support site. -->
+Download the `.dcos` package of the DC/OS Monitoring Service from Mesosphere support site.
+
+
+## Install the service
+
+<!-- At what point is the "dcos registry add" command added? -->
```
Comment (gracedo) [inline pages/services/dcos-monitoring/0.2.1/install/index.md]: @SarahStegall The command is listed [below](https://github.com/mesosphere/dcos-docs-site/pull/1788/files#diff-1d8c85a2425b1692cfd1fe9cc9b1e4a4R33) before the `package install` command - is that sufficient?

File: pages/services/dcos-monitoring/0.2.1/install/index.md [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+
+## Install package registry
+
+Please follow these [instructions](https://docs.mesosphere.com/1.12/administering-clusters/repo/package-registry/quickstart/) to install the package registry.
+
+# Install DC/OS Monitoring service
+
+## Download the package
+
+<!-- Does this require that the user have a service account? Also, please add a link to the support site. -->
```
Comment (gracedo) [inline pages/services/dcos-monitoring/0.2.1/install/index.md]: @SarahStegall Will add link to support site. It looks like the [above link to package registry installaion](https://github.com/mesosphere/dcos-docs-site/pull/1788/files#diff-1d8c85a2425b1692cfd1fe9cc9b1e4a4R17) indicates when a service account is necessary (for DC/OS enterprise) and instructions of how to add one, so perhaps that link is sufficient?

Comment (gracedo) [reply pages/services/dcos-monitoring/0.2.1/install/index.md]: sorry missed this - it looks like service account instructions are also [provided below](https://github.com/mesosphere/dcos-docs-site/pull/1788/files#diff-1d8c85a2425b1692cfd1fe9cc9b1e4a4R51)

**Comment by gracedo** [comment]
@SarahStegall If you're all good with my changes, can we get this merged? Thanks!

**Pull request merged**